src/api/controllers/routes.py

```diff
@@ -137,30 +172,4 @@ def create_app() -> Flask:
             app.logger.exception("Error %s occurred. Exiting...", str(e))
             sys.exit(1)
 
-    # This version value is substituted dynamically at build time
-    app.config["VERSION"] = "Unknown"
-
-    # Flask has some Anys in its type stubs, so the lines relating to the api object
-    # have to have comments to suppress mypy errors.
-
-    # Register healthz and version endpoints
-    api.add_resource(Ready, "/healthz/ready")  # type: ignore[misc]
-    api.add_resource(Live, "/healthz/live")  # type: ignore[misc]
-    api.add_resource(Version, "/version")  # type: ignore[misc]
-
-    # Register Zones endpoints
-    api.add_resource(ZoneListResource, "/zones")  # type: ignore[misc]
-    api.add_resource(ZoneDescribeResource, "/zones/<zone_name>")  # type: ignore[misc]
-
-    # Register Criticalservices endpoints
-    api.add_resource(CriticalServiceListResource, "/criticalservices")  # type: ignore[misc]
-    api.add_resource(  # type: ignore[misc]
-        CriticalServiceDescribeResource, "/criticalservices/<service_name>"
-    )
-    api.add_resource(CriticalServiceUpdateResource, "/criticalservices")  # type: ignore[misc]
-    api.add_resource(CriticalServiceStatusListResource, "/criticalservices/status")  # type: ignore[misc]
-    api.add_resource(  # type: ignore[misc]
-        CriticalServiceStatusDescribeResource, "/criticalservices/status/<service_name>"
-    )
-
     return app
```

Comment:
Oh, I just noticed that this ends up returning the app that was created. In that case, probably a change would be needed, because I suspect you'd want this function to return without waiting for RR to be enabled. Otherwise I'm not sure that the API server would actually be running.